src/EnergyPlus/BaseboardElectric.cc

```diff
@@ -361,24 +361,29 @@ namespace BaseboardElectric {
                                 "Baseboard Total Heating Energy",
                                 OutputProcessor::Unit::J,
                                 thisBaseboard.Energy,
-                                "System",
-                                "Sum",
+                                OutputProcessor::eTimeStepType::System,
+                                OutputProcessor::eVariableType::Sum,
```

Comment:
Nearly all the changes in this PR are going from strings to enums on these two arguments.  I wrote a script that did this pretty intelligently and ended up getting no diffs at all.